netapp_dataops_traditional/netapp_dataops/clone_refresh_and_protect.sh

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
Just as an FYI - I will be deleting this script from the main repo after merging the PR. We have an internal repo that we store test scripts in, so I will be storing it there.

Reply:
done ...